lib/ScrollControls/ScrollControls.js

```diff
@@ -105,6 +103,9 @@ var ScrollControls = module.exports = kind(
 		}
 		if (this.scroller) {
 			this.scroller.on('scrollabilityChanged', this._updateScrollability);
+			this.subControls.forEach(this.bindSafely(function (nom) {
```

Comment:
Is `subControls` intended to be overridden by something? If not, I don't think we're gaining anything with the array and then iterating over it. `forEach` isn't particularly performant as well so, if we need the array, we should probably swap it out for a `for` loop instead.